app/views/users/dashboard_sidebar.html.erb

```diff
@@ -41,10 +41,12 @@
 <% end %>
 <div>
   <% if show_user_create_course_button(@current_user) %>
+    <% if @show_feature %>
```

Comment:
Are we intentionally leaving this in or was this only supposed to be a test of the gem functionality?

Reply:
should be removed.